presto-docs/src/main/sphinx/release/release-0.263.rst

```diff
@@ -0,0 +1,23 @@
+=============
+Release 0.263
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a correctness bug in distinct aggregations when combined with a filtering mask containing null values.
+* Improve spilling CPU and memory efficiency for distinct and order by aggregations.
+* Add a new session property ``heap_dump_on_exceeded_memory_limit_enabled`` to enable heapdump on exceeded memory failures. The heapdump file directory can be provided using ``exceeded_memory_limit_heap_dump_file_directory`` session property.
+* Added Smart Sampling feature that samples the tables at query (planning) time. This helps in cases when the user is just exploring the data, testing a pipeline or looking for trends. We sample all the tables (and only tables) in such a way that all "key-based" operations like JOIN/GROUP BY/window functions etc will be complete for the sampled keys.
```

Comment:
@kaikalur Any config/session properties?  Can you please reference those properties' documentation and make this less verbose?

Reply:
just say `Add` instead of `Added` .
like  \``JOIN\`` /  \``GROUP BY``

Reply:
* Added Smart Sampling feature that samples tables at query time. This feature can be enabled using session property ``key_based_sampling_enabled=true``. Additionally, sampling rate and sampling function name can be provided by sessions properties ``key_based_sampling_percentage`` and ``key_based_sampling_function`` respectively.